plugins/inputs/cisco_telemetry_mdt/cisco_telemetry_mdt.go

```diff
@@ -782,27 +798,10 @@ func (c *CiscoTelemetryMDT) parseContentField(
 	delete(tags, prefix)
 }
 
-func (c *CiscoTelemetryMDT) Address() net.Addr {
+func (c *CiscoTelemetryMDT) address() net.Addr {
 	return c.listener.Addr()
 }
```

Comment:
Why do we need this function? But maybe we shouldn't remove it in this PR. What do you think?

Reply:
We don't. Removed.